app/presenters/tree_builder_miq_action_category.rb

```diff
@@ -0,0 +1,53 @@
+class TreeBuilderMiqActionCategory < TreeBuilder
+  has_kids_for Classification, [:x_get_tree_classification_kids]
+
+  private
+
+  def override(node, object, _pid, _options)
+    img = "blank.gif"
+    leaf = !object.entries.any?
+    img = "tag.png" unless leaf
```

Comment:
Please try to always keep the whole image name in one, place, `100/#{img}` means that if you search for `100/tag.png`, you won't find it :)

Also, `blank.gif` is no longer needed, simple `nil` would suffice.

So, I'd propose something like

```
image = leaf ? nil : "100/tag.png"
node[:image] = image && ActionController::Base.helpers.image_path(image)
```